### ISSUE TITLE
Make some of the bot error messages nicer for beginners

### ISSUE THREAD

**Issue by mazalkov** [python-discord/meta#162]
### Amendment type

Content Update

### Content type

Other

### Please provide the location of the content

https://github.com/python-discord/bot/blob/main/bot/constants.py#L766

### What amendment do you think should be made to this content, and why?

Have seen people in the Discord try to use the bot, then get greeted with error messages such as:

_Please don't do that._
_You have to stop._
_In the future, don't do that._
_You blew it._
_You're bad at computers._

They're pretty funny, sure, but people who are new to Python and/or the Advent of Code tend to get frustrated with the bot and give up altogether. Already seen a few people stop interacting with the server after this.

Maybe just make some of those error messages a bit nicer so people who are new to the Discord/Python in general aren't put off by the condescending/sarcastic tone?

**Comment by onerandomusername** [comment]
This is also a problem in Sir-Lancebot, and I have to agree.

When I was newer, getting error messages from Lance felt too mean, and it doesn't help that Lance and bot are so liberal with their error messages.

for example, using a command in the wrong channel is greated with an error message that then deletes itself.

Some of the other error messages feel out of place in pydis, like "Are you trying to kill me?"

I do think it would be worth revising the error messages and taming them down a bit.

**Comment by Sn4u** [comment]
I like the sass. Gives the bot some personality

**Comment by onerandomusername** [comment]
I'm not opposed to the feature itself, but there's a few messages that feel unwelcoming, like they go against the CoC.

I don't think that there should be an message that can't be said to a user by other users, eg "you're bad at computers". I would hate having someone tell me that when starting out, so I feel like Lance shouldn't say it either.

**Comment by dementati** [comment]
I think there's a middle ground here where we don't have to lose what gives the bot personality while at the same time ensuring that the messages are in fact funny and not easily interpreted as condescending and sarcastic.

**Comment by DMFriends** [comment]
Here's my opinion on this.

I personally see both sides to this. From what I understand from the original post, it seems like people simply took the error messages too seriously, which I don't think was the right thing to do on their part. After all, the error messages are all randomized and you never know which one will come up the next time. So I think it should be fine either way.

**Comment by mbaruh** [comment]
I tend to agree with most of what was said in the OP, the messages are too blunt.

Like dem said we don't need to strip the personality, the arguement is against this specific kind of messages.

**Comment by onerandomusername** [comment]
Yeah, its not like we need to yank the feature entirely, that was never the intention. 

We can even brainstorm some more ideas, and use those.

But messages like "You're bad at computers" feel like the CoC wouldn't allow a normal user to say that, which does not make sense for a *bot* to be able to tell that to users.

**Comment by ramii19780529** [comment]
As an experienced developer, I am so bored with normal error messages. I would fall out of my seat if Oracle said "You fool, that's VARCHAR2, not NUMBER."

That being said, for new developers, I can see how they might be upset with seeing messages like that went trying to learn something new. On the other hand, it's a bot error message, not a real person, and someone giving up because a bot hurt their feelings is a bit ridiculous imo.

**Comment by minalike** [comment]
We do plan on executing this, it's just been delayed.
